lib/sna/auth/email.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Sna.Auth.Mail do
+
+  @moduledoc """
+
+  Sna.Auth.Mail will store email and token:
+
+  1. when an user comes and want to use email authentication, an email
+     is firstly checked.
+
+  2. if the mail is valid, a token is generated and the email is
+     stored temporarily in the ETS cache. A timer process is started
+     for some minutes.
+
+  3. if the token is valid (checked by the user), the mail is stored
+     in the underlaying database else, after some minutes, the mail
+     and token are rejected.
+
+  4. after validation, Sna.Auth.Mail process will return the session
+     identifier.
```

Comment:
Session management belongs to SnaWeb, no ?

Reply:
Session management can be managed anywhere in the code, actually, SnaWeb is only the snippet from Phoenix to generate everything. When I was coding with cowboy in erlang, the session can be managed in other place. But you are right, maybe it could be a good idea to move that to SnaWeb, except if we want to split these feature and allow different authentication method in a near future (e.g. ueberauth).